.eslintrc.js

```diff
@@ -279,6 +279,7 @@ module.exports = {
     'tooling/*',
     '**/test/**/*',
     'docs/**',
+    '*.json',
```

Comment:
Added this because eslint was giving me a "Missing semicolon" error for `cspell.json`.